why-apollo-advantages-disadvantages-alternatives.md

```diff
@@ -23,25 +23,25 @@ summary = "It is not always simple to pick the right solution for a problem. In
+While GraphQL is in its early stages, the Apollo ecosystem offers solutions for many of its challenges.  Beyond that we can see how much the ecosystem is growing, because the company announces an update for Apollo or another library that can be used with Apollo's tech stack at every other technology conference.  Apollo isn't just covering GraphQL, though; they also have effort invested in REST interfaces for backward compatibility to RESTful architectures.  This takes GraphQL beyond the network layer and remote data by offering a state management solution for local data, too. 
 
 {{% sub_chapter_header "The Company and Community behind Apollo" "apollo-company-community" %}}
 
-The company behind Apollo (Meteor Development Group Inc.) is pouring lots of resources into its endeavour. They are active in open source, write lots of articles about their product, and are seen often at conferences. In general, the GraphQL ecosystem {{% a_blank "seems to be in good shape for the future" "https://techcrunch.com/2018/05/15/prisma" %}}. Many people are speaking about it and are keen to adopt it for their own applications. In addition, even though Apollo is powered by its company, the community behind it is growing. More people are adopting GraphQL and opt-in Apollo for their client-side and server-side JavaScript applications.
+The company behind Apollo--Meteor Development Group Inc.--is pouring lots of resources into its success. Meteor is also active in open source, offering in-depth articles about their products, supported by an established presence at the conferences. In general, the GraphQL ecosystem {{% a_blank "seems to be in good shape for the future" "https://techcrunch.com/2018/05/15/prisma" %}}. The community behind Graph QL is growing, as more developers are adopt it and use Apollo for client and server-side JavaScript applications.
 
 {{% sub_chapter_header "Who is using Apollo?" "apollo-who-is-using" %}}
 
-There are plenty of popular and tech savvy companies out there who are using Apollo already. Not only the Apollo company itself, who was developing the popular Meteor framework before, but also companies such as Airbnb and Twitch are using it. The following list is an excerpt of companies and their stories using Apollo for their JavaScript applications:
+Tech-savvy companies are taking advantage of Apollo already. Many were familiar popular Meteor framework from before, but new and extremely popular companies like Airbnb and Twitch are using it. These are just a few of their stories:
```

Comment:
"Many were familiar popular Meteor framework from before, but" -> "Not only the Apollo company itself, who was developing the popular Meteor framework before, but"

Reply:
Not a good sentence.  We already know Apollo is using its own software and that they developed Meteor from earlier.  Even if you don't say it, it's obvious.  The point is that other companies are using it, which is why I started the sentence "Many other companies"

Reply:
Okay 👍